preview/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM buildpack-deps:trusty-scm
+
+RUN apt-get update \
+    && wget https://dotnetcli.blob.core.windows.net/dotnet/beta/Installers/Latest/dotnet-ubuntu-x64.latest.deb \
```

Comment:
I think it is worth mentioning why the package is not retrieved from the Debian repository.  I think this may be a source of confusion as already observed in previous comments on this checkin.